draft-ietf-rats-architecture.md

```diff
@@ -1095,16 +1095,45 @@ Attestation Result.  In this case the epoch is said to be rough because:
 * The time between the creation of Claims and the collection of Claims is
   indistinguishable.
 
+## Implicit Timekeeping using Epoch Handles
+
+A third approach relies on having epoch "handles"
+periodically sent to both the sender and receiver of Evidence or
+Attestation Results by some "Handle Distributor".
+
+Handles are different from nonces as they can be used more than once and
+can even be used by more than one entity at the same time.
+Handles are different from timestamps as they do not have to convey information about a point in time.
+
+Like the nonce approach, this allows associating a "rough" epoch without
+requiring a reliable clock or time synchronization in order to generate or
+appraise the freshness of Evidence or Attestation Results.  Only the
+Handle Distributor requires access to a clock so it can periodically send
+new epoch handles.
+
+The most recent handle is included in the produced Evidence or Attestation
+Results, and the appraising entity can compare the handle in received
+Evidence or Attestation Results against the latest handle it received from
+the Handle Distributor to determine if it is within the current epoch.
+An actual solution also needs to take into account race conditions
+when transitioning to a new epoch, such as by using a counter signed
+by the Handle Distributor as the handle, or by including both the current and
```

Comment:
Updated text per Thomas's comment.   If the handle is a signed counter, and the handle is sent in (say) evidence, then the verifier receiving it that hasn't seen the new handle yet can simply accept it if it's signed by the HD it trusts and it sees the counter is higher than the handle it has.